package.json

```diff
@@ -65,22 +67,16 @@
       "salesforce-alm"
     ],
     "devPlugins": [
-      "@oclif/plugin-command-snapshot"
+      "@oclif/plugin-command-snapshot",
+      "@salesforce/plugin-release-management"
     ],
     "update": {
       "s3": {
         "bucket": "dfc-data-production",
         "indexVersionLimit": 20,
         "folder": "media/salesforce-cli/sfdx",
         "acl": " ",
-        "host": "https://developer.salesforce.com",
```

Comment:
where did all this functionality go?

Reply:
@mshanemc this configuration hasn't been needed since we moved to the new pack/upload/promote commands in oclif

Reply:
I think we are using it 
https://github.com/oclif/oclif/search?q=s3.host

Reply:
if that's true, could it be removed from here? https://github.com/oclif/oclif/blob/711132e6a29ba82e370a5c90cc5059682c389a0f/src/upload-util.ts#L23

Reply:
I believe there's a WI for that already